frontend/public/components/secrets/create-secret.tsx

```diff
@@ -908,6 +928,20 @@ export const EditSecret = ({match: {params}, kind}) => <Firehose resources={[{ki
   <SecretLoadingWrapper fixedKeys={['kind', 'metadata']} titleVerb="Edit" saveButtonText="Save" />
 </Firehose>;
 
+export type SecretLoadingWrapperProps = {
+  obj?: {
+    [key: string]: Object;
+  };
+  fixedKeys: string[];
+  titleVerb: string,
+  saveButtonText: string,
+};
+
+export type SecretLoadingWrapperState = {
+  formComponent: any;
```

Comment:
```suggestion
  formComponent: React.ReactNode;
```

Reply:
Actually this needs to be `React.ReactType`